src/components/OverlayPanel.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useCallback, useReducer } from "react";
+import { styled } from "@storybook/theming";
+
+import OverlayPortal from "./OverlayPortal";
+import OverlayButtons from "./OverlayButtons";
+import OverlayImage from "./OverlayImage";
+
+interface OverlayPanelProps {
+    image_url: string;
+}
+
+interface OverlayState {
+    showOverlay: boolean;
+    lockOverlay: boolean;
+    showDifference: boolean;
+    overlayScaling: number;
+    opacity: number;
+}
+
+const initialState: OverlayState = {
+    showOverlay: false,
+    lockOverlay: false,
+    showDifference: false,
+    overlayScaling: 0.5,
```

Comment:
I think default one should be x1 and we should be able to scale it down wit x0.5.

Reply:
Oh good to know! So, for now, we can keep a predefined set of scalings instead of putting more effort into adding configurable parameters? And use the API response in a future change?

Reply:
Agreed